docs/02_getting-started/03_smart-contract-development/01_hello-world.md

```diff
@@ -42,7 +42,6 @@ using namespace eosio;
 
 class [[eosio::contract]] hello : public contract {
   public:
-  	 using contract::contract;
```

Comment:
this is redundant with what line? I've tried to build the eosio.cdt hello example without this line and I couldn't, it failed compilation.

Reply:
hi @iamveritas  this is a draft PL. Not ready for review yet. Also, see the [comment](https://github.com/EOSIO/welcome/pull/201#issuecomment-619760974) I made earlier

Reply:
I got email saying `@dixia requested your review on: #201 [draft PR] updated out-of-date description.`
that's why I looked at it. and I checked the files changed yes.

Reply:
no worry. 

could you try to use the code in `01_hello-world.md` to test? The change is based on the code from that file

also, which version of CDT are you using?